features/step_definitions/back_office/registration_steps.rb

```diff
@@ -48,7 +48,6 @@
   info_panel = @bo.registration_details_page.info_panel
   page_content = @bo.registration_details_page.content
   expect(@bo.registration_details_page.business_name).to have_text(@business_name)
-  expect(@bo.registration_details_page.content).to have_text("Bob Carolgees")
```

Comment:
I removed this as it was too specific and I don't think is super important to keep. Refactoring this to make it still work (hence check that the content contains the name of a person registered) I think is too much work for little gain in this case. Happy to discuss it :)

Reply:
That's fine :) There's a check on `@business_name` which is the most important thing.